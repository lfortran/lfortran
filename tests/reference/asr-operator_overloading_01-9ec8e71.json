{
    "basename": "asr-operator_overloading_01-9ec8e71",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/operator_overloading_01.f90",
    "infile_hash": "f44f3172b4b5235b41deb5a4d4eaadc61a6b7c51e002a983889cf38f",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-operator_overloading_01-9ec8e71.stdout",
    "stdout_hash": "cda4d65c84ff3da92bfe4c20c89b06fd3d7b520cd521a05c36c84845",
    "stderr": "asr-operator_overloading_01-9ec8e71.stderr",
    "stderr_hash": "bc887b577bc8ccfc15f212c070a67ee8c67af8d343abdd0132e6b6fb",
    "returncode": 0
}