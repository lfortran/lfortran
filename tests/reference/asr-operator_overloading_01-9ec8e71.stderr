style suggestion: Use 'end if' instead of 'endif'
  --> tests/../integration_tests/operator_overloading_01.f90:28:9
   |
28 |         endif
   |         ^^^^^ help: write this as 'end if'
